astropy/units/tests/test_quantity_non_ufuncs.py

```diff
@@ -1500,13 +1500,13 @@ class TestSortFunctions(InvariantUnitTestSetup):
     def test_sort(self):
         self.check(np.sort)
 
+    def test_sort_axis(self):
+        self.check(np.sort, axis=0)
+
     @needs_array_function
     def test_sort_complex(self):
         self.check(np.sort_complex)
 
-    def test_msort(self):
-        self.check(np.msort)
```

Comment:
Since, this is only deprecated in the latest dev version of `numpy` shouldn't this check remain in place until we have dropped support for numpy < 1.24?

Reply:
done